pygmt/base_plotting.py

```diff
@@ -1983,3 +1983,82 @@ def update_pointers(data_pointers):
             with file_context as fname:
                 arg_str = " ".join([fname, build_arg_string(kwargs)])
                 lib.call_module("meca", arg_str)
+
+    @fmt_docstring
+    @use_alias(
+        R="region",
+        J="projection",
+        G="fill",
+        B="frame",
+        Td="day_night",
+        Tc="civil_twighlight",
+        Tn="nautical_twighlight",
+        Ta="astronomical_twighlight",
+        W="pen",
+        U="timestamp",
+        V="verbose",
+        X="xshift",
+        Y="yshift",
+        p="perspective",
+    )
+    @kwargs_to_strings(R="sequence", p="sequence")
+    def solar(self, **kwargs):
+        r"""
+        A plotting function the day-night terminator. Additionally, it can
+        plot the terminators for civil twighlight, nautical twighlight, and
```

Comment:
Is twighlight the British spelling? The GMT docs at https://docs.generic-mapping-tools.org/6.1/solar.html uses twilight.

Reply:
Nope, I just spelled it incorrectly once and didn't realize my mistake and kept copying it. Fixing it.

Even if it was, I'm an American just living in the UK, so I have no excuse.